huaweicloud/services/lts/resource_huaweicloud_lts_structuring_custom_configuration.go

```diff
@@ -0,0 +1,308 @@
+// ---------------------------------------------------------------
+// *** AUTO GENERATED CODE ***
+// @Product LTS
+// ---------------------------------------------------------------
+
+package lts
+
+import (
+	"context"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+const (
+	RegexParseType = "custom_regex"
+	JsonParseType  = "json"
+	SplitParseType = "split"
+	NginxParseType = "nginx"
```

Comment:
```suggestion
	regexParseType = "custom_regex"
	jsonParseType  = "json"
	splitParseType = "split"
	nginxParseType = "nginx"
```
only constants are used in this package.

Reply:
OK